larsim/PhotonPropagation/PhotonVisibilityService.h

```diff
@@ -298,6 +300,27 @@ namespace phot {
 
     MappedFunctions_t doGetTimingTF1(geo::Point_t const& p) const;
 
+    void findVoxelSuggestion(std::string& logString,
+                             float tpcMin,
+                             float tpcMax,
+                             float cryoMin,
+                             float cryoMax,
+                             int& nVoxels,
+                             float& voxelMin,
+                             float& voxelMax,
+                             float voxelSizeGoal = 10.0) const;
+
+    float testVoxelSuggestion(std::string& logString,
+                              float tpcMin,
+                              float tpcMax,
+                              float cryoMin,
+                              float cryoMax,
+                              int& nVoxels,
+                              float& voxelMin,
+                              float& voxelMax,
+                              float voxelSizeGoal = 10.0,
+                              int jog = 0) const;
```

Comment:
I suggest the following adjustment:

```suggestion
    float testVoxelSuggestion(float tpcMin,
                              float tpcMax,
                              float cryoMin,
                              float cryoMax,
                              int& nVoxels,
                              float& voxelMin,
                              float& voxelMax,
                              float voxelSizeGoal,
                              int jog,
                              std::string* logString = nullptr) const;
```

The `voxelSizeGoal` and `job` arguments are always specified when `testVoxelSuggestion(...) is invoked.  And by making `logString` a pointer and giving it a default value of `nullptr`, you can avoid passing in dummy strings when there's nothing to log.

Reply:
Legacy of development, good call. Fixed.